src/js/main.js

```diff
@@ -159,6 +147,19 @@ window.addEventListener("touchend", () => {
 	mouse.dragCallback = mouse.endCallback = false;
 });
 
+window.onmousemove = e => {
+	mouse.x = e.clientX;
+	mouse.y = e.clientY;
+	if (mouse.dragCallback) {
+		e.preventDefault();
+		mouse.dragCallback(e);
+	}
+};
+window.onmouseup = () => {
```

Comment:
While you're at it, maybe be DRY and merge the `touchend` and `onmouseup` code? (Possibly also for `touchmove` and `onmousemove`?)

And maybe use `addEventListener` for both types of events, to make code more consistent?